Frameworks/CoreImage/CIImage.mm

```diff
@@ -73,90 +74,164 @@ @implementation CIImage
 /**
  @Status Interoperable
 */
-- (CIImage*)init {
++ (CIImage*)emptyImage {
+    return [[[CIImage alloc] init] autorelease];
+}
+
+/**
+ @Status Interoperable
+*/
+- (CGRect)extent {
+    return self->_extent;
```

Comment:
Convention dictates that we never use `self->` in ObjC code unless it's unclear. It's just an instance variable.

Also, this could be done via `@property`; you wouldn't need to implement `-extent` yourself